examples/dotnet/FunctionExecutionCs/README.md

```diff
@@ -0,0 +1,25 @@
+﻿# FunctionExecution Example
+This example illustrates how to execute server side java functions.
+
+## Prerequisites
+* Install [Apache Geode](https://geode.apache.org)
+* Build and install [Apache Geode Native](https://github.com/apache/geode-native)
+
+## Running
+* Run the following Powershell script which starts the Geode Locator, deploys the jar file containing your function, start the Geode Server, and create region.
+  ```
+  PS> startserver.ps1
+  ```
+* Execute `Apache.Geode.Examples.FunctionExecutionCs.exe`.
+  
+  output:
+  ```
+  Storing id and username in the region
+  Getting the user info from the region
+  rtimmons = Robert Timmons
+  scharles = Sylvia Charles
+  Function Execution Results:
```

Comment:
I don't think the output looks like this, anymore. Line 75 just prints the number of elements in the result set, nothing more.

Reply:
Did u run it?

Reply:
It wouldn't be what's in the README. The print from line 75 is missing, and I don't see where lines 21 and on (in the README) would come from. `Console.WriteLine("Result count of: " + res.Count)`

Reply:
quite certain that was generated output and then added to the README...

Reply:
Looks like the last change wasn't checked in. New commit forthcoming.

Reply:
Please update README.md with the correct output:

```
C:\Program Files\nativeclient\examples\build\dotnet\FunctionExecutionCs\Debug>FunctionExecutionCs.exe
Storing id and username in the region
Getting the user info from the region
rtimmons = Robert Timmons
scharles = Sylvia Charles
Result count of: 1

```

Reply:
README.md has been updated with correct output.